sm_property_tree/CMakeLists.txt

```diff
@@ -32,6 +32,10 @@ target_link_libraries(${PROJECT_NAME}
 # Avoid clash with tr1::tuple: https://code.google.com/p/googletest/source/browse/trunk/README?r=589#257
 add_definitions(-DGTEST_USE_OWN_TR1_TUPLE=0)
 
+#TODO(Hannes) remove this as soon as catkin includes tools/threads.cmake again (this would make exactly the following implicitly)
+find_package(Threads REQUIRED)
+SET(THREADS_LIBRARY ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
No, actually it is not required transitively - in contrast to some boost libs.

It might even just be needed for the gtests below (probably due to a catkin bug, see https://github.com/ros/catkin/issues/745).

I decided against removing it up there and fixing it later just for the gtest because as soon as catkin gets fixed its not needed anymore at all, if it is only needed for gtest - otherwise this spot is the right place anyways.